test/builder/QuarkBuilderCometWithdraw.t.sol

```diff
@@ -545,7 +546,7 @@ contract QuarkBuilderCometWithdrawTest is Test, QuarkBuilderTest {
 
         QuarkBuilder builder = new QuarkBuilder();
 
-        vm.expectRevert(QuarkBuilder.MaxCostTooHigh.selector);
+        vm.expectRevert(abi.encodeWithSelector(Actions.NotEnoughFundsToBridge.selector, "usdc", 99e6, 99e6));
```

Comment:
Since this error were reachable in some test cases and some are not is due to....
some has smarter bridging logics on payment token, and some didn't have it. (Bad things happened to repeating copying code pattern turned out to not having consistent behavior :P) and which is why I decided to prioritize refactor tasks haha.  (Cuz this might become a bug in the future)

So with the bridging payment token attempt logics,  the error will reached with `NotEnoughFundsToBridge` instead of MaxCostTooHigh.  MaxCostTooHigh is only reachable if we don't include the logics that try to bridge payment token.

Reply:
Also raised the `note` here whether or not to ditch this check: 
https://github.com/compound-finance/quark-scripts/blob/a90666e69cb0c40292e500ac888e3192c08cc365/src/builder/QuarkBuilderBase.sol#L230